Schweizerisches Bundesgericht.js

```diff
@@ -0,0 +1,243 @@
+{
+	"translatorID": "9e898223-938c-4c8f-8b5c-d118b6e44aae",
+	"label": "Schweizerisches Bundesgericht",
+	"creator": "Hans-Peter Oeri",
+	"target": "^https://www\\.bger\\.ch/ext/eurospider/live/(de|fr|it)/php/(clir|aza)/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-11-13 21:17:33"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2020 Hans-Peter Oeri
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+const abbrevPublished = {
+	de: "BGE ",
+	fr: "ATF ",
+	it: "ATF "
+};
+
+const abbrevUnpublished = {
+	de: "BGer ",
+	fr: "TF ",
+	it: "TF "
+};
+
+const courtName = {
+	de: "Schweizerisches Bundesgericht",
+	fr: "Tribunal Fédéral Suisse",
+	it: "Tribunale Federale Svizzero"
+}
+
+/**
+ * read query parameters from url
+ *
+ * @param   {String} url
+ * @returns {{}}
+ */
+function parseURLParameters(url) {
+	let parts = require('url').parse(url, true);
+	parts.query._lang = parts.pathname.substring(21, 23);
```

Comment:
Can this be a regex? Feels unstable.

Reply:
changed to path particle.

(NB the site is 20 years old (and it shows) - if they change something, everything will break)